app/controllers/relationships_controller.rb

```diff
@@ -0,0 +1,36 @@
+class RelationshipsController < ApplicationController
+  before_action :logged_in_user
+  before_action :find_user, only: :create
+  before_action :find_relationship, only: :destroy
+
+  def create
+    current_user.follow @user if @user.present?
```

Comment:
```suggestion
    if current_user.follow @user
    ...
    else
    ...
    end
```
không cần check `@user.present?` nữa nhé, đã check ở find_user rồi